homeassistant/components/mqtt/__init__.py

```diff
@@ -187,7 +188,11 @@
     required=True,
 )
 
-CONFIG_SCHEMA_BASE = vol.Schema(
+PLATFORM_CONFIG_SCHEMA_BASE = vol.Schema(
+    {vol.Optional(component.value): cv.ensure_list for component in PLATFORMS}
```

Comment:
This PR only adds support for `fan` and `light` so I think we should do like this for now:
```suggestion
    {
        vol.Optional(Platform.FAN.value): cv.ensure_list,
        vol.Optional(Platform.LIGHT.value): cv.ensure_list,
    }
```